### ISSUE TITLE
Arch build instructions

### ISSUE THREAD

**Issue by swaptr** [ElementsProject/lightning#5215]
I am trying to build lightning from source on Arch linux. There don't seem to be any instructions listed specific to arch based distros [here](https://github.com/ElementsProject/lightning/blob/master/doc/INSTALL.md). A lot of people seem to use Arch based distros and it would be a good idea to list some instructions. Thoughts?

**Comment by vincenzopalazzo** [comment]
>I am trying to build lightning from source on Arch linux. There don't seem to be any instructions listed specific to arch-based distros [here](https://github.com/ElementsProject/lightning/blob/master/doc/INSTALL.md). A lot of people seem to use Arch-based distros and it would be a good idea to list some instructions. Thoughts?

You are right, we miss it but due to my last configuration with arch Linux, I was able to compile core lightning with no additional dependencies (just `pip install poetry`)

I will take this task, at list add some information about the os, in the meanwhile can you try to compile it and report some error if any?

**Comment by swaptr** [comment]
Sure, will do.

**Comment by swaptr** [comment]
I tried compiling lightning but it gives `ERROR: git is required for generating version information`.

**Comment by vincenzopalazzo** [comment]
This will be fixed with this https://github.com/ElementsProject/lightning/pull/5210

The solution is to run the following command on the root of the directory

`git config --system --add safe.directory 'pwd'/*'`

P.S: override the ' with ` around pwd in the terminal!

**Comment by sangbida** [comment]
This issue is being closed as part of issue cleanup.

**Reason**: No recent activity (last updated 1388 da

If you believe this issue should remain open, please comment and we can reopen it.
